central/cve/csv/handler.go

```diff
@@ -118,92 +126,204 @@ func CVECSVHandler() http.HandlerFunc {
 			csv.WriteError(w, http.StatusBadRequest, err)
 			return
 		}
+		rawQuery, paginatedQuery := resolvers.V1RawQueryAsResolverQuery(rQuery)
 
-		ctx, err := csvHandler.GetScopeContext(loaders.WithLoaderContext(r.Context()), query)
+		cveRows, err := CVECSVRows(loaders.WithLoaderContext(r.Context()), query, rawQuery, paginatedQuery)
 		if err != nil {
 			csv.WriteError(w, http.StatusInternalServerError, err)
-			log.Errorf("unable to determine resource scope for query %q: %v", query.String(), err)
 			return
 		}
 
-		rawQuery, paginatedQuery := resolvers.V1RawQueryAsResolverQuery(rQuery)
+		postSortRequired := paginatedQuery.Pagination == nil ||
+			paginatedQuery.Pagination.SortOption == nil ||
+			paginatedQuery.Pagination.SortOption.Field == nil
+
+		output := newCSVResults(csvHeader, postSortRequired)
+		for _, row := range cveRows {
+			output.addRow(row)
+		}
+		output.Write(w, "cve_export")
```

Comment:
Augment the filename with timestamp- `cve_export_<<ts>>`

Reply:
Since adding timestamp is not mentioned in changelog, would it affect the users calling the API through their automated workflows?